forms-flow-web/src/components/Form/Steps/PreviewStepper.js

```diff
@@ -419,7 +419,7 @@ const Preview = React.memo(
                           "Applicable for Client and Reviewer Roled Users only."
                         )}
                       >
-                        {t("Client Permission")}
+                        {t("Permission to create new submission")}
```

Comment:
I think this won't be very clear.  because the same auth handler manage view submission of a form right ?

Reply:
I think it's fine with this label; as anyone would expect to see their submissions and it is implicit as standard.